platform-coop-toolkit/pinecone
# feat: page graphics

## Description

Integrate visual designs into Pinecone.

## Steps to test

1. Review layouts:

- https://deploy-preview-294--pinecone.netlify.com/components/detail/home.html
- https://deploy-preview-294--pinecone.netlify.com/components/detail/page.html
- https://deploy-preview-294--pinecone.netlify.com/components/detail/archive--default.html
- https://deploy-preview-294--pinecone.netlify.com/components/detail/search-results--default.html
- https://deploy-preview-294--pinecone.netlify.com/components/detail/favorites.html
- https://deploy-preview-294--pinecone.netlify.com/components/detail/term-list--topics.html
- https://deploy-preview-294--pinecone.netlify.com/components/detail/term-list--goals.html
- https://deploy-preview-294--pinecone.netlify.com/components/detail/term-list--sectors.html
- https://deploy-preview-294--pinecone.netlify.com/components/detail/term-list--coop-types.html
- https://deploy-preview-294--pinecone.netlify.com/components/detail/term-list--formats.html

**Expected behavior:** Observe visual designs.

## Additional information

Not applicable.

## Related issues

- https://github.com/platform-coop-toolkit/coop-library/pull/222
- https://github.com/platform-coop-toolkit/coop-library/issues/224
- https://github.com/platform-coop-toolkit/coop-library/issues/231
